src/grequests_transparency_portal_cities.py

```diff
@@ -0,0 +1,93 @@
+import os
```

Comment:
This script (`src/grequests_transparency_portal_cities.py`) has to be documented in `CONTRIBUTING.md` before merging, ok?